plugins/nodes/src/js/pages/NodesPage.js

```diff
@@ -3,10 +3,18 @@ import React from "react";
 import { Icon } from "@dcos/ui-kit";
 import { ProductIcons } from "@dcos/ui-kit/dist/packages/icons/dist/product-icons-enum";
 import { iconSizeS } from "@dcos/ui-kit/dist/packages/design-tokens/build/js/designTokens";
+import { Helmet } from "react-helmet";
 
 class NodesPage extends React.Component {
   render() {
-    return this.props.children;
+    return (
+      <React.Fragment>
+        <Helmet>
+          <title>{i18nMark("Nodes")}</title>
```

Comment:
I don't think we ever show just this page. Do we need a title?

Reply:
I would still keep it.